Makefile

```diff
@@ -1,4 +1,5 @@
-SYMBOLIC_PYTHON := python3
+SHELL = /bin/bash
+export SYMBOLIC_PYTHON_VERSION := python3
```

Comment:
this is unused now?

Reply:
The `SHELL` is probably unnecessary; I did have to add the `export` to get `venv` to work.

Reply:
Sorry, I think I misunderstood you. @jan-auer has pointed out that I forgot to use the `$SYMBOLIC_PYTHON_VERSION` variable.

Reply:
Oh, my example was for when you define it without the `export`, but you can keep that too. So what do we need the `SHELL` for?

Reply:
I don't think it's needed for anything, to be honest. Initially I couldn't get it to work and just copied the first two lines from the symbolicator Makefile. I'll remove it.